ultraScurve.py

```diff
@@ -14,8 +14,6 @@
                   help="Specify Output Filename", metavar="filename")
 parser.add_option("--latency", type="int", dest = "latency", default = 37,
                   help="Specify Latency", metavar="latency")
-parser.add_option("--mspl", type="int", dest = "MSPL", default = 4,
```

Comment:
All of the scan scripts had this option hard coded in each of the scan options.  So I moved it to qcoptions.py

See lines 3-4 of:

https://github.com/bdorney/vfatqc-python-scripts/blob/develop/qcoptions.py